CHANGELOG.md

```diff
@@ -131,6 +131,8 @@
 
 * `filter/subset_obsp` component: Added a component to subset an .obsp matrix by column based on the value of an .obs field. The resulting subset is moved to an .obsm field (PR #888).
 
+* `labels_transfer/knn` component: Allow to perform KNN classification based on a pre-calculated neighborhood graph (PR #890).
```

Comment:
On line 45 of the CHANGELOG there is a an entry for the knn component to be removed. Could you merge the entry that you just created with this one? i.e. how the knn component has changed compared to before #830 Does this PR revert the changes from #830?